Neos.Fusion/Classes/FusionObjects/RendererImplementation.php

```diff
@@ -20,51 +20,10 @@
  */
 class RendererImplementation extends AbstractFusionObject
 {
+    use RendererTrait;
 
-    /**
-     * The type to render if condition is true
-     *
-     * @return string
-     */
-    public function getType()
```

Comment:
those getters are pretty much useless.

while calling in eel `${this.type}` will actually result in the getter being called - instead of using the arrayAccess functionality of the fusionObject. In this case 100% the same code is executed: `$this->fusionValue('type');`
```
root >
root = Neos.Fusion:Renderer {
    renderer = ${this.type}
}
```
sometimes  we do some type castings in the getters, but that can also be done inline.
Also we are always quite forgiving (by casting) so that a possible return type will almost never provoke any (then descriptive) errors.